### ISSUE TITLE
Publish Java artefacts to Maven Central?

### ISSUE THREAD

**Issue by anjackson** [iipc/urlcanon#3]
Are the Java Maven artefacts published anywhere? Can we get them published to Maven Central? Both myself and @johnerikhalse have credentials for publishing there via [SonaType OSSRH](http://central.sonatype.org/pages/ossrh-guide.html), so we could help with this?

**Comment by anjackson** [comment]
Ah, I see it's already set up with references to OSSRH in the pom. So maybe I'm really asking if you are planning to make a release?

**Comment by ato** [comment]
Ah, sorry for the delay. I've released the current code as 0.1.0. It should be available as soon as central syncs:

```xml
<dependency>
  <groupId>org.netpreserve</groupId>
  <artifactId>urlcanon</artifactId>
  <version>0.1.0</version>
</dependency>
```

Note that not all unit tests pass yet. There's a few places where the Java code is not quite at feature parity with the Python code (IPv6 normalisation, unicode character corner cases not supported by the Java standard library). Some of the other tests I'm suspicious whether are correct behaviour but I haven't had a chance to go through them against the spec yet. Nevertheless for the majority of URLs the library is already quite useful and we're (NLA) using it in production in a few places already.
